sdk/android-sdk/src/main/java/eu/intent/sdk/api/ITMessagingApi.java

```diff
@@ -148,6 +148,16 @@ public void updateMessage(String messageId, String newTitle, String newText, ITA
     }
 
     /**
+     * Gets the number of unread messages for the given equipment and activity category.
+     *
+     * @param equipmentId         the ID of an ITEquipment
+     * @param activityCategoryIds the IDs of several ITActivityCategories
+     */
+    public void getUnreadMessagesCountByEquipmentIdAndActivityCategory(String equipmentId, List<String> activityCategoryIds, ITApiCallback<ITMessagesCount.ByActivityCategory> callback) {
```

Comment:
Peut-être une incohérence entre le commentaire + le nom, et ce que fait la fonction : 
`for the given equipment and activity category` -> on a l'impression que c'est 1 id de category à passer en argument, et pas une collection de catégories

Pareil pour le nommage de la fonction, j'aurais mis `…AndActivityCategory` -> `…AndActivityCategories`